src/main/kotlin/hs/kr/equus/user/domain/auth/service/QueryPassInfoService.kt

```diff
@@ -0,0 +1,45 @@
+package hs.kr.equus.user.domain.auth.service
+
+import hs.kr.equus.user.domain.auth.entity.PassInfo
+import hs.kr.equus.user.domain.auth.entity.repository.PassInfoRepository
+import hs.kr.equus.user.domain.auth.exception.InvalidPassException
+import hs.kr.equus.user.domain.auth.presentation.dto.resopnse.QueryPassInfoResponse
+import hs.kr.equus.user.global.utils.pass.PassUtil
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+
+@Service
+class QueryPassInfoService(
+    private val passInfoRepository: PassInfoRepository,
+    private val passUtil: PassUtil
+) {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line